content/departments/product-engineering/process/sdlc.md

```diff
@@ -2,33 +2,37 @@
 
 <span class="badge badge-note">SOC2/GN-98</span>
 
-Sourcegraph uses three types of approaches to drive changes:
+Sourcegraph uses three types of approaches to drive changes (either a new feature or extending an existing one).
```

Comment:
Changes are not always features though?

Reply:
You're right, error on my end, I'll correct that!